python_modules/dagster/dagster/_core/definitions/decorators/asset_decorator.py

```diff
@@ -1119,9 +1120,22 @@ def inner(fn: Callable) -> AssetsDefinition:
             input_name: asset_key for asset_key, (input_name, _) in asset_ins.items()
         }
         asset_outs = build_asset_outs(outs)
+
+        check_specs_by_output_name = _validate_and_assign_output_names_to_check_specs(
+            check_specs, list(asset_outs.keys())
+        )
+        check_outs_by_output_name: Mapping[str, GraphOut] = {
+            output_name: GraphOut() for output_name in check_specs_by_output_name.keys()
+        }
+
+        combined_outs_by_output_name = {
+            **{output_name: GraphOut() for output_name, _ in asset_outs.values()},
```

Comment:
do we not need to shuffle any information (e.g. description) to the graph out?

Reply:
Good catch, we don't for the asset outs either which seems like a bug. https://github.com/dagster-io/dagster/pull/16490/files#diff-0fcad2d82f20f3987322def0a04db6e04bc81b9777e5ced50f0fed5e527872bdL1124

It also looks like we don't set a description on graph outs anywhere in the code base. I want to do this in a follow up just on the off chance that we don't want to pick it in to the release

Reply:
actually it's unclear if graph outs get surfaced anywhere

Reply:
Did this in https://github.com/dagster-io/dagster/pull/16504